smartcontractkit/external-adapters-js
# update eth gas station api url

## Closes 28404

## Description

The ETH Gas Station API has been returning errors due to the EA making calls to a bad endpoint.  This PR updates the endpoint to the correct one.

## Changes

- Update endpoint called within the EthGasStation EA



## Steps to Test

1. Run EA and send request below
```
{
  "jobRunID": "1",
  "data": {
      "endpoint":"gasprice",
      "speed":"fast"
    }
}
```
2. Verify successful response

## Quality Assurance
